priv/repo/migrations/20170504190323_create_tasks.exs

```diff
@@ -0,0 +1,24 @@
+defmodule CoursePlanner.Repo.Migrations.CreateTasks do
+  use Ecto.Migration
+
+  def change do
+    execute("""
+          CREATE TYPE task_status AS ENUM (
+            'Pending',
+            'Accomplished'
+          )
+    """)
+    create table(:tasks) do
+      add :name, :string
+      add :deadline, :date
+      add :status, :task_status
```

Comment:
I am also not sure about this status, if the task is not something to be delivered, maybe those statuses don't 
make sense